extensions/admin-tool-ext-win/src/main.ts

```diff
@@ -7,82 +7,87 @@
-
-        const serverdownloader = new ServerProvider(config);
-        const installationStart = Date.now();
-
-        try {
-            let downloadedExePath = await serverdownloader.getOrDownloadServer();
-            const installationComplete = Date.now();
-
-            // Don't register the command if we couldn't find the EXE since it won't be able to do anything
-            if(downloadedExePath) {
-                exePath = downloadedExePath;
-            } else {
-                throw new Error('Could not find SsmsMin.exe after downloading');
-            }
-            // Add the command now that we have the exePath to run the tool with
-            context.subscriptions.push(
-                vscode.commands.registerCommand('adminToolExtWin.launchSsmsServerPropertiesDialog', handleLaunchSsmsServerPropertiesDialogCommand));
-
-            Telemetry.sendTelemetryEvent('startup/ExtensionStarted', {
-                installationTime: String(installationComplete - installationStart),
-                beginningTimestamp: String(installationStart)
-            });
-        }
-        catch(err) {
-            Telemetry.sendTelemetryEvent('startup/ExtensionInitializationFailed', {
-                error: err
-            });
-        }
-    } else {
-        vscode.window.showErrorMessage(localize('adminToolExtWin.onlySupportedOnWindows', 'The Admin Tool Extension is only supported on Windows platforms.'));
-    }
+	// This is for Windows-specific support so do nothing on other platforms
+	if (process.platform === 'win32') {
+		Telemetry.sendTelemetryEvent('startup/ExtensionActivated');
+
+		let config: IConfig = JSON.parse(JSON.stringify(baseConfig));
+		config.installDirectory = path.join(context.extensionPath, config.installDirectory);
+		config.proxy = utils.getConfiguration('http').get('proxy');
+		config.strictSSL = utils.getConfiguration('http').get('proxyStrictSSL') || true;
+
+		const serverdownloader = new ServerProvider(config);
+		const installationStart = Date.now();
+
+		try {
+			let downloadedExePath = await serverdownloader.getOrDownloadServer();
+			const installationComplete = Date.now();
+
+			// Don't register the command if we couldn't find the EXE since it won't be able to do anything
+			if (downloadedExePath) {
+				exePath = downloadedExePath;
+			} else {
+				throw new Error('Could not find SsmsMin.exe after downloading');
+			}
+			// Add the command now that we have the exePath to run the tool with
+			context.subscriptions.push(
+				vscode.commands.registerCommand('adminToolExtWin.launchSsmsServerPropertiesDialog', handleLaunchSsmsServerPropertiesDialogCommand));
+
+			Telemetry.sendTelemetryEvent('startup/ExtensionStarted', {
+				installationTime: String(installationComplete - installationStart),
+				beginningTimestamp: String(installationStart)
+			});
+		}
+		catch (err) {
+			Telemetry.sendTelemetryEvent('startup/ExtensionInitializationFailed', {
+				error: err
```

Comment:
this won't send any callstacks or error messages that could values we don't to capture?

Reply:
I need to follow up on this - but for now I'll remove this and just send the event alone.